lazy_tensor_core/lazy_tensor_core/csrc/op_by_op_executor.cpp

```diff
@@ -52,7 +52,7 @@ lazy_tensors::hash_t ComputeNodeKey(
                  GetParameterShape(operands[i], *input_shapes[i])));
   }
   key = lazy_tensors::util::HashCombine(
-      key, lazy_tensors::util::ShapeHash(node->shape()));
+      key, lazy_tensors::util::ShapeHash(GetShapeFromTsNode(*node)));
```

Comment:
each backend will override their hashing or will we standardize on what `shape()` from the base class return?